### PR TITLE
[jiralert] Fix template error in ingress

### LINKED ISSUE #3929: [jiralert] Missing service Configuration Causes Template Rendering Error
### Describe the bug a clear and concise description of what the bug is.

When trying to use the Jiralert Helm chart, I encountered a template rendering error due to a missing service configuration in the values.yaml file.

### What's your helm version?

v3.13.1

### What's your kubectl version?

Client Version: v1.28.3 Server Version: v1.20.11

### Which chart?

Jiralert

### What's the chart version?

1.6.0

### What happened?

`Error Message: Error: template: jiralert/templates/ingress.yaml:3:23: executing "jiralert/templates/ingress.yaml" at <.Values.service.port>: nil pointer evaluating interface {}.port`

The `values.yaml` file doesn't seem to have the service configuration, which is causing the above error when Helm tries to render the ingress.yaml template.

### What you expected to happen?

The values.yaml file should ideally contain a service section to prevent such template rendering issues and to ensure the Ingress setup works correctly.

### How to reproduce it?

_No response_

### Enter the changed values of values.yaml?

service:
  port: 9097

### Enter the command that you execute and failing/misfunctioning.

kustomize build . --enable-helm

### Anything else we need to know?

I am using Kustomize to generate the k8s manifests.

### PR DESCRIPTION
**Pull request by jkroepke**
#### Which issue this PR fixes

*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*

- fixes #3929

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

File: charts/jiralert/ci/test-values.yaml [hunk 1]
```diff
@@ -5,3 +5,6 @@ config:
   receivers:
     - name: 'default'
       project: EXAMPLE
+
+ingress:
+  enabled: true
```
Comment (zanhsieh) [inline charts/jiralert/ci/test-values.yaml:10]: Why enabled by default?

File: charts/jiralert/templates/ingress.yaml [hunk 2]
```diff
@@ -49,10 +48,10 @@ spec:
               service:
                 name: {{ $fullName }}
                 port:
-                  number: {{ $svcPort }}
+                  number: 9097
               {{- else }}
               serviceName: {{ $fullName }}
-              servicePort: {{ $svcPort }}
+              servicePort: 9097
```
Comment (zanhsieh) [inline charts/jiralert/templates/ingress.yaml:54]: Why hardcoded?

**Review by zanhsieh — CHANGES_REQUESTED** [review]

File: charts/jiralert/ci/test-values.yaml (hunk 1 above)
Comment (jkroepke) [reply charts/jiralert/ci/test-values.yaml:10]: Recheck the context. This is the test values file and ingress should be covered by tests

File: charts/jiralert/templates/ingress.yaml (hunk 2 above)
Comment (jkroepke) [reply charts/jiralert/templates/ingress.yaml:54]: Because the service is hardcoded to. Whats the reason to have to configurable?

Comment (zanhsieh) [reply charts/jiralert/templates/ingress.yaml:54]: What if the users would like to config themselves?

Comment (jkroepke) [reply charts/jiralert/templates/ingress.yaml:54]: I'm not aware of any use-case for it.

Comment (jkroepke) [reply charts/jiralert/templates/ingress.yaml:54]: Please mention, the service https://github.com/prometheus-community/helm-charts/blob/84ab6297ac99330cf3dc3d9c68d811172582acaa/charts/jiralert/templates/service.yaml#L10-L14 is hardcoded for over one year and no one needs to just it. I guess it was from the old days, there hostport was required to expose services.

**Review by zanhsieh — APPROVED** [review]

**Comment by jkroepke** [comment]
blocked by https://github.com/prometheus-community/helm-charts/pull/3962

**Pull request merged**